custom_components/kia_uvo/Vehicle.py

```diff
@@ -173,7 +173,7 @@ def set_last_updated(self):
             hour=int(m.group(4)),
             minute=int(m.group(5)),
             second=int(m.group(6)),
-            tzinfo=local_timezone,
+            tzinfo=dt_util.get_time_zone("UTC"),
```

Comment:
This is not valid for EU because response from kia servers return time in local time, not in UTC. Can you confirm response for CA in UTC? in any case, let's separate these 2 changes as sensors.py is a quick bug fix, which we introduced recently.

Reply:
Yes CA is in UTC.  Before this change my updates were reporting as in the future.

Reply:
is it fine for you if you can separate this into another PR?